src/guides/v2.3/frontend-dev-guide/templates/template-email-migration.md

```diff
@@ -0,0 +1,230 @@
+---
+group: frontend-developer-guide
+title: Migrating custom email templates
+functional_areas:
+  - Frontend
+---
+
+{% raw %}
+With the release of Magento 2.3.4, we made some changes to custom email templates and how they access data and methods for email content.
+This topic describes the changes and provides instructions on how to convert your existing custom email templates.
+
+## Changes to the custom email template workflow
+
+In Magento 2.3.4, custom email templates are only allowed to use scalar values for variable data.
```

Comment:
As of Magento 2.3.4, 

(I'm just leaving a comment here to determine whether I'm experiencing problems posting review comments everywhere, or just in a certain other PR.)

Reply:
Done